inference_experimental/inference_exp/models/auto_loaders/auto_negotiation.py

```diff
@@ -174,6 +193,7 @@ def determine_default_allowed_quantization(
             return [
                 Quantization.UNKNOWN,
                 Quantization.FP32,
+                Quantization.FP16,
```

Comment:
```suggestion
```

delete because we only added for florence 2, but we changed package to register as unknown instead since it decides at runtime based on cpu/gpu env